controllers/tab/settings/masthead/form/MastheadForm.inc.php

```diff
@@ -30,11 +30,9 @@ function MastheadForm($wizardMode = false) {
 			'printIssn' => 'string',
 			'onlineIssn' => 'string',
 			'description' => 'string',
-			'mailingAddress' => 'string',
-			'journalEnabled' => 'bool',
-			'categories' => 'object',
 			'masthead' => 'string',
-			'history' => 'string'
+			'aboutJournal' => 'string',
```

Comment:
Rather than `aboutJournal` in OJS and `aboutPress` in OMP, why not consolidate this into a setting called `about` and implement it in `lib-pkp`?